### ISSUE TITLE
Cannot execute code actions

### ISSUE THREAD

**Issue by janivo** [TypeFox/monaco-languageclient#295]
Hey there,

im currently using this package to glue together an `omnisharp-roslyn` (C#) language server and the `monaco-editor` running in an electron frontend. 

I'm struggling to get any commands working which involves the communication between the lang-server and the editor. Everything works fine (rendering, diagnostics, decorations, etc.) except the code actions.

In beforehand, I receive this object as part of the registration request which should automatically lead to registration inside the `MonacoServices` (no?):
```json
{
    "id": "e965316d-9701-4977-9feb-ff32fc0478be",
    "method": "workspace/executeCommand",
    "registerOptions": {
        "commands": [
            "omnisharp/executeCodeAction"
        ]
    }
}
```

The action are generated by the lang-server and displayed correctly:
![image](https://user-images.githubusercontent.com/22963121/136869030-4816dd42-7846-4937-862b-486f170ad6e0.png)

... but as soon as I try to run some action within the bulb menu, I get this error:
```
command '[object Object]' not found
  | notify | @ | simpleServices.js:151
  | error | @ | simpleServices.js:146
  | (anonymous) | @ | codeActionCommands.js:145
  | rejected | @ | codeActionCommands.js:18
  | Promise.then (async)
  ....
```
After some debugging I found out that the nesting of the `codeActionItem` does not match the expected format and in general seems kinda off, no?
```
{
    "command": {
        "id": {
            "title": "Mars.Interfaces.Data.LayerInitData",
            "command": "omnisharp/executeCodeAction",
            "arguments": [
                {
                    "Uri": "file:///PAHT_TO_MY_FILE",
                    "Identifier": "Mars.Interfaces.Data.LayerInitData",
                    "Name": "Mars.Interfaces.Data.LayerInitData",
                    "Range": {
                        "Start": {
                            "Line": 10,
                            "Character": 43
                        },
                        "End": {
                            "Line": 10,
                            "Character": 43
                        }
                    }
                }
            ]
        },
        "title": "Mars.Interfaces.Data.LayerInitData"
    },
    "title": "Mars.Interfaces.Data.LayerInitData"
}
```
I don't really have the big picture yet, where I can pin point to the breaking location, so I wanted to ask you what the problem might be. It is some kind of version mismatch? Because it seems like some component isn't really handling the command correctly. Here are my dependencies in case it might be important:
```
├─ @codingame/monaco-jsonrpc@0.3.1
├─ monaco-editor-webpack-plugin@5.0.0
├─ monaco-editor@0.29.1
├─ monaco-languageclient@0.13.1-next.9
├─ vscode-jsonrpc@6.0.0
├─ vscode-languageclient@7.0.0
├─ vscode-languageserver-protocol@3.16.0
├─ vscode-languageserver@7.0.0
```

**Comment by janivo** [comment]
First of all, thank you for your fast response!
And second of all, **that really did the trick**. Thank you very much, you've saved me from a lot of headaches!

On a sidenote: since when and why did the scope of the `monaco-languageclient` change to `@codingame`?

**Comment by janivo** [comment]
Yes, I do. But this is more of an inconvenience for the user, so it's not a priority for me at the moment. 

But thank you for the heads-up, I'll keep an eye on this issue.

**Comment by CGNonofr** [comment]
Be careful though, because the handlers are binded twice, and only unbinded once when you destroy the LanguageClient.

This leads to many errors if you stop or restart the LanguageClient.

**Comment by janivo** [comment]
Thank you for the hint. Haven't had any trouble with this yet.

Speaking of starting/stopping the LanguageClient. Right now I see no way to add/remove files dynamically once the language server has started. Is there a way to implement this kind of feature with the `monaco-languageclient` to tell omnisharp that there are new/less files? Right now, I'm restarting the whole server when adding/removing files which feels very wrong.
